example/usps_api/server.cc

```diff
@@ -0,0 +1,43 @@
+#include <string>
+#include <iostream>
+
+#include <grpc/grpc.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+#include <grpcpp/security/server_credentials.h>
+
+/* https://grpc.io/docs/languages/cpp/basics/
+ * Runs server using server builder
+ */ 
```

Comment:
I know the style guide said it's fine to use either /* */ or //, but // is more commonly used, so let's change this to the following:

```
// https://grpc.io/docs/languages/cpp/basics/
// Runs server using server builder
```

Reply:
Done